physionet-django/user/views.py

```diff
@@ -448,6 +448,7 @@ def auth_orcid(request):
         orcid_profile.token_type = token.get('token_type')
         orcid_profile.token_scope = token.get('scope')
         orcid_profile.token_expiration = token.get('expires_at')
+        orcid_profile.date_added = datetime.fromtimestamp(token.get('expires_at')) - timedelta(days=20*365.2422)
```

Comment:
We used the expiry date to guess the `date_added` when it wasn't known. As we are now capturing the date directly, can't this just be date_added = now? I think if you use `auto_now_add=True` in the model, it may not be necessary to modify the view at all.

Reply:
Please see my previous comment which also addresses this. I'm open to changing this if desired though.